modules/microshift-adding-service-to-blueprint.adoc

```diff
@@ -2,16 +2,21 @@
 //
 // microshift/microshift-embed-into-rpm-ostree.adoc
 
-:_content-type: CONCEPT
+:_content-type: PROCEDURE
 [id="adding-microshift-service_{context}"]
 = Example Image Builder blueprint with {product-title}
 
 Adding the {product-title} RPM package to an Image Builder blueprint enables the build of a {op-system-ostree} image with {product-title} embedded.
 
+.Procedure
+
+. Use the following example to create your blueprint:
+
 .Image Builder blueprint example
 
 [source,text]
 ----
+$ sudo cat > minimal-microshift.toml <<EOF
```

Comment:
This command has the same questions about ownership and using cat vs. tee as above.